### PR TITLE
ab: facs draw histogram version 1

### LINKED ISSUE #573: AB: FACS draw histogram
Depends on #571 

from https://www.dropbox.com/s/5dy257x6e4kc5sb/SCB_ProblemSetBuilder_09_FlowCyt.pdf?dl=0

image

### PR DESCRIPTION
**Pull request by annagav**
Fix #573 

Add drawing functionality for FACS analyze page.

### CODE REVIEW

File: instructor/common.py [hunk 1]
```diff
@@ -1144,58 +1144,48 @@ def create_facs_histograms(assignment, facs_sample):
...
         ]
         if sample.fixed:
-            grouped_forms.append(('fixed', sample.analysis, sample.condition, form_list))
+            grouped_histograms.append(('fixed', sample.analysis, sample.condition, form_list))
         if sample.live:
-            grouped_forms.append(('live', sample.analysis, sample.condition, form_list))
+            grouped_histograms.append(('live', sample.analysis, sample.condition, form_list))
+
+    histogram_data = {}
+    instances = models.FlowCytometryHistogram.objects.filter(sample_prep__assignment=assignment)
```
Comment (justinabrahms) [inline instructor/common.py]: Why don't use use the instances variable from above on line 1155?

Comment (annagav) [inline instructor/common.py]: oops. That was the plan :D

**Comment by justinabrahms** [comment]
Looks code looks pretty good to me. I haven't demoed the functionality, yet. How do I get to this page?

**Comment by annagav** [comment]
You have to go to all the pages defining everything for your assignment.

**Comment by justinabrahms** [comment]
:+1:

**Pull request merged**